server/src/main/java/org/atmosphere/nettosphere/BridgeRuntime.java

```diff
@@ -123,6 +118,7 @@
 @Sharable
 public class BridgeRuntime extends HttpStaticFileServerHandler {
 
+    private final ExecutorService executor = Executors.newSingleThreadExecutor();
```

Comment:
I think we should make this optional, and the Thread pool configuration and optional. Making it the default change the current behaviour, and we will have to bump to a major version `4.0.0`